src/lib/components/informationBox/index.tsx

```diff
@@ -19,47 +22,55 @@ const InformationBox = ({
   title,
   children,
   showIcon,
-  size = 'default'
-}: InformationBoxProps) => (
-  <div
-    className={classNames(
-      className, 
-      'p16 br8 color-black', 
-      styles.informationBox,
-      styles[`informationBox--${variant}`]
-    )}
-    role="alert"
-  >
-    <div className='d-flex'>
-      {showIcon && 
-        <span
-          data-testid="information-box-icon"
-          className={classNames(
-            styles.icon,
-            styles[`icon--${variant}`],
-            'mr8'
-          )}
-        />
-      }
-      <div>
-        {title && (
-          <h4 
-            data-testid="information-box-title"
-            className={classNames(
-              size === 'default'? 'p-h4' : 'p-h5',
-              'mb8'
-            )}
-          >
-            {title}
-          </h4>
+  size = 'default',
+}: InformationBoxProps) => {
+  const mapAlertIconColor: { [k in Variant]: Color } = {
+    warning: 'yellow-500',
+    error: 'red-500',
+    success: 'green-500',
+    information: 'blue-500',
+    neutral: 'purple-500',
+  };
+
+  return (
+    <div
+      className={classNames(
+        className,
+        'p16 br8 color-black',
+        styles.informationBox,
+        styles[`informationBox--${variant}`]
+      )}
+      role="alert"
+    >
+      <div className="d-flex">
+        {showIcon && (
+          <div data-testid="information-box-icon" className={'mr8'}>
+            <AlertCircleIcon
+              className={styles.icon}
+              color={mapAlertIconColor[variant]}
```

Comment:
I think this was wrong before, can you add size 24 if props.size is default and 16 when small?